chevah/compat/tests/elevated/test_system_users.py

```diff
@@ -244,6 +244,84 @@ def test_authenticateWithUsernameAndPassword_good(self):
         else:
             self.assertIsNotNone(token)
 
+    @conditionals.onOSFamily('posix')
+    def test_authenticateWithUsernameAndPassword_passwd_valid(self):
```

Comment:
Is there a reason for not changing the test method name to reflect the actual method tested?